user-management.yaml

```
@@ -238,6 +245,13 @@ paths:
         - uc4_auth: [] 
       description: "Returns all students, may only be used by administrators"
```

is this still true, or can all fetch users using a list of usernames, similar to above?
```suggestion
      description: ""Without parameter \"usernames\", returns all students, and may only be used by administrators. With \"usernames\" set, returns all students with the given usernames, which can be invoked by all roles.""
```